docs/atlascli/command/atlas-clusters-onlineArchives-create.txt

```diff
@@ -43,32 +43,50 @@ Options
      - Description
    * - --archiveAfter
      - int
-     - true
+     - false
      - Number of days after which to archive cluster data.
+
+       Mutually exclusive with --file.
    * - --clusterName
      - string
      - true
      - Name of the cluster.
    * - --collection
      - string
-     - true
+     - false
      - Name of the collection.
+
+       Mutually exclusive with --file.
    * - --dateField
      - string
-     - true
+     - false
      - Name of an already indexed date field from the documents.
+
+       Mutually exclusive with --file.
    * - --dateFormat
      - string
      - false
-     - Date format for the date field. Valid values are ISODATE, EPOCH_SECONDS, EPOCH_MILLIS, or EPOCH_NANOSECONDS. This value defaults to "ISODATE".
+     - Date format for the date field. Valid values are ISODATE, EPOCH_SECONDS, EPOCH_MILLIS, or EPOCH_NANOSECONDS.
+
+       Mutually exclusive with --file. This value defaults to "ISODATE".
```

Comment:
Suggest removing quotes to match above.

Reply:
unfortunately I can't remove the quotes given it comes from the tooling, to make things consistent I've added quotes on the large text

Reply:
reminder the tooling is owned by us (APIx and docs) https://github.com/mongodb-labs/cobra2snooty contributions are welcome